app/kubemci/pkg/gcp/forwardingrule/forwardingrulesyncer.go

```diff
@@ -17,9 +17,14 @@ package forwardingrule
 import (
 	"fmt"
 	"net/http"
+	"reflect"
+	"sort"
+
+	compute "google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
 
```

Comment:
Why this empty line? Remove?

Reply:
Done. no idea. I just let emacs do its thing. I assume it's from whatever auto-generates the imports.